### ISSUE TITLE
Amazon S3 downloads broken / missing

### ISSUE THREAD

**Issue by davemitchell** [jruby/jruby#6729]
https://s3.amazonaws.com/jruby.org/downloads/1.7.27/jruby_windows_x64_1_7_27.exe (as well as earlier versions) is broken, returning 

<Error>
  <Code>NoSuchKey</Code>
  <Message>The specified key does not exist.</Message>
  <Key>downloads/1.7.27/jruby_windows_x64_1_7_27.exe</Key>
  <RequestId>8B7Q3NHSBY2TNVND</RequestId>
  <HostId>I6jrdoWrjdkJXwb+niKNNufBqpDjiS9nt/B2HKUjM/Z6T4LSQpYQhKyqchaZjW0ctLCgwBptCYs=</HostId>
</Error>

**Comment by headius** [comment]
@enebo I think these exes are long lost now, yes?

@davemitchell Is it a problem to upgrade to JRuby 9.2?

**Comment by enebo** [comment]
@davemitchell there was a snafu and we thought S3 was backing up keys and someone deleted out buckets by accident.  This particular installer was one of the things we did not have a local back up of.

I am curious how you found this particular link.  It was a valid link but our website probably has a link in the old 1.7.27 accouncement page perhaps?

**Comment by davemitchell** [comment]
https://www.jruby.org/files/downloads/1.7.27/index.html. (via https://www.jruby.org/download and "all releases")

This is a legacy project, based on ruby 1.9.3-p551 and using jruby 1.7.25 to run on a windows server that's been under maintenance for a while and will be replaced by year's end (entirely new software and stack).  It turns out that v1.7.25 was apparently linked with an older version of openssl, and about a week ago a number of heroku-based sites started rejecting TLS v1.0 & v1.1 causing a number of integrations to stop working.  jruby 1.7.27 looks to have been issued some time later with a later version of openssl that supports more resent TLS protocols.  Everything works under 1.7.27 from a mac, but when I tried to update the windows version, the installer wasn't available.  Any suggestions?  1.7 => 9xxx is probably too much of an upgrade to undertake for this project which has limited days, and not worth the extended effort for re-testing production critical functionality.

**Comment by davemitchell** [comment]
ps, I believe 1.7.27 was the last in the line of 1.7 before the switch to 9.x.   It looks like the other archives on that page (bin, src, and complete) are available still available.  Would any of them contain the same content that could be manually installed on a windows server?

**Comment by headius** [comment]
@davemitchell You are correct, 1.7.27 is the last 1.7 release. The other archives are there I believe because we had those files saved in other places, while the .exe was only published to S3.

The contents of the .exe installer are mostly just the contents of the bin zip file plus the installation logic (unpack it in the right place, set up env vars like PATH, maybe a few other little things). It would be quite feasible to just do those steps manually (or through an automated process other than our installer).
